src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java

```diff
@@ -15,6 +15,7 @@
 import com.unstoppabledomains.resolution.dns.DnsRecordsType;
```

Comment:
@DancingAxolotl 

Could you mint two same domains on L1 and L2 and test specific behaviour for this case? L2 should have a precedence.

Reply:
Good idea. I haven't considered such a case.